<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,5,1,3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="ip_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 5px;
background-color: rgb(153, 193, 241);
border-color: rgb(222, 221, 218);</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>opposite_ip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="port_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 5px;
background-color: rgb(143, 240, 164);
border-color: rgb(222, 221, 218);</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>opposite_port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="self_recv_port_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 5px;
background-color: rgb(249, 240, 107);
border-color: rgb(222, 221, 218);</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>self_recv_port</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="connect_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;
border-color: rgb(222, 221, 218);
background-color: rgb(46, 194, 126);</string>
       </property>
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;
border-color: rgb(222, 221, 218);
background-color: rgb(255, 190, 111);</string>
       </property>
       <property name="text">
        <string>recivied</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="send_Edit">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;
border-color: rgb(222, 221, 218);
background-color: rgb(246, 97, 81);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QTextEdit" name="recivied_Edit">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;
border-color: rgb(222, 221, 218);
background-color: rgb(255, 190, 111);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;
border-color: rgb(222, 221, 218);
background-color: rgb(246, 97, 81);</string>
       </property>
       <property name="text">
        <string>send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,1">
     <item>
      <widget class="QLineEdit" name="send_message_edit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;
border-color: rgb(222, 221, 218);
background-color: rgb(255, 120, 0);</string>
       </property>
       <property name="placeholderText">
        <string extracomment="self_port"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="send_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;
border-color: rgb(222, 221, 218);
background-color: rgb(224, 27, 36);</string>
       </property>
       <property name="text">
        <string>send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="forward_Button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 5px;
border-color: rgb(222, 221, 218);
background-color: rgb(248, 228, 92);</string>
         </property>
         <property name="text">
          <string>forward</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="left_Button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 5px;
border-color: rgb(222, 221, 218);
background-color: rgb(87, 227, 137);</string>
           </property>
           <property name="text">
            <string>left</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="right_Button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 5px;
border-color: rgb(222, 221, 218);
background-color: rgb(98, 160, 234);</string>
           </property>
           <property name="text">
            <string>right</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="back_Button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 5px;
border-color: rgb(222, 221, 218);
background-color: rgb(192, 97, 203);</string>
         </property>
         <property name="text">
          <string>back</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="rocker_bar" name="rocker" native="true">
       <property name="styleSheet">
        <string notr="true">#rocker
{
	border-radius: 5px;
	border-color: rgb(222, 221, 218);
	background-color: rgb(220, 138, 221);
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>rocker_bar</class>
   <extends>QWidget</extends>
   <header location="global">rocker_bar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
